configs/raspberrypi_config

```diff
@@ -66,6 +66,7 @@ function platform_modify() {
   run_and_log_cmd "sudo cp work/rpi/binaries/u-boot.bin work/boot/${RASPBERRYPI_KERNEL_IMAGE}"
   run_and_log_cmd "sudo cp work/rpi/binaries/boot.scr work/boot"
   run_and_log_cmd "sudo cp work/rpi/binaries/fw_env.config work/rootfs/etc/"
+  run_and_log_cmd "sudo cp work/rpi/binaries/uboot-git-log.txt work/boot"
```

Comment:
Did you regenerate the binares for Raspberry Pi to include this as well?

Could you also update the commit message with a bit more background information on why?

Reply:
I did not regenerate them but I can.  Should we use a version number there as well?

Reply:
No need for this iteration, so just drop above change

Reply:
If I drop it here, chances are good it will not happen any time soon and it is a very handy thing to have given the propensity of cloudfront to cache the files.
I'm ok not pushing new binaries but let's at least leave the commit in the tree for future use.  I realize we don't have it for the other platforms either but I don't have any of them so I didn't test.  I _did_ test the build for RpI, just didn't publish new binaries.

Reply:
You will have to push new binaries, otherwise this command will fail.

Reply:
Ahh right.  I have removed it.